Client/Frontend/Settings/AppSettingsTableViewController.swift

```diff
@@ -57,6 +58,12 @@ class AppSettingsTableViewController: SettingsTableViewController, FeatureFlagsP
         case .customizeToolbar:
             let viewModel = SearchBarSettingsViewModel(prefs: profile.prefs)
             viewController = SearchBarSettingsViewController(viewModel: viewModel)
+
+        case .wallpaper:
+            let viewModel = WallpaperSettingsViewModel(with: tabManager, and: WallpaperManager())
```

Comment:
Can we have an explanation here since .wallpaper has different logic than the other cases?